### PR TITLE
Fix imports, add ONNX exports

### PR DESCRIPTION
**Pull request by k0nze**
…error.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -2,3 +2,5 @@ numpy~=1.15.0
 torch~=1.0.0
 torchvision~=0.2.0
 visdom~=0.1.0
+Pillow==6.1
+onnx==1.6.0
```
Comment (activatedgeek) [inline requirements.txt]: `torch`, `torchvision` and `visdom` packages here are probably too old. Would you mind fixing them to the latest major.minor versions and update to something like `torch>=1.4.0`.

File: run.py [hunk 2]
```diff
@@ -83,9 +84,15 @@ def train_and_test(epoch):
...
+    dummy_input = torch.randn(1, 1, 32, 32, requires_grad=True)
+    torch.onnx.export(net, dummy_input, "lenet.onnx")
+
+    onnx_model = onnx.load("lenet.onnx")
+    onnx.checker.check_model(onnx_model)
+
 
 def main():
-    for e in range(1, 16):
+    for e in range(1, 3):
```
Comment (activatedgeek) [inline run.py]: Does it cross 99% right after 3 epochs? If not do you mind keeping this the same.

Comment (k0nze) [reply run.py]: No, it doesn't I reduced the epochs for testing. I'm going to revert that.

Comment (k0nze) [reply run.py]: Now after letting it train for 16 epochs it reaches an accuracy of: 0.990800. With the added residual connection and the updated packages in `requirements.txt`

**Comment by k0nze** [comment]
It Now reaches an accuracy > 0.99 with the residual connection and the updated package versions in the `requirements.txt`

**Comment by activatedgeek** [comment]
Thank you!
